src/scene-entry-manager.js

```diff
@@ -213,6 +214,28 @@ export default class SceneEntryManager {
       spawnMediaInfrontOfPlayer(e.detail, contentOrigin);
     });
 
+    this.scene.addEventListener("object_pinned", e => {
+      const el = e.detail.el;
+      const networkId = el.components.networked.data.networkId;
+      const gltfNode = pinnedEntityToGltf(el);
```

Comment:
Hmm I wonder if it makes more sense to just send the data over to reticulum and have it do this part. Seems a bit odd that the wire protocol is gltf nodes... But maybe thats nice?

Reply:
yeah I feel like this is the proper separation of concerns. it's nice that reticulum can remain completely agnostic to aframe and networked-aframe schema.